frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/AnticipatoryActionStormPanel/index.tsx

```diff
@@ -54,6 +75,33 @@ function AnticipatoryActionStormPanel() {
         <div className={classes.titleSelectWrapper}>
           <Typography variant="h2">{t('STORM - Global view')}</Typography>
         </div>
+        <div className={classes.titleSelectWrapper}>
+          <div className={classes.titleSelectWrapper}>
```

Comment:
Do we need the duplicate div with the same class?

Reply:
fixed